.github/workflows/release.yaml

```diff
@@ -61,3 +64,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           COSIGN_EXPERIMENTAL: true
+
+      - name: Push latest label for core
```

Comment:
I know this PR is fixing a pain we have, but I'd rather have this as part of goreleaser instead.

Reply:
Then review https://github.com/open-telemetry/opentelemetry-collector-releases/pull/220

Reply:
Done ;-)

Reply:
@jpkrohling review again please :)